README.md

```diff
@@ -233,7 +233,7 @@ _Related: [Polls and Events](#polls-and-events)_
 - [dyu bookmarks](https://github.com/dyu/bookmarks) - Single-threaded/process bookmark app powered by leveldb and uWebSockets. Supports importing from Delicious and Chrome. ([Demo](https://dyuproject.com/bookmarks/)) `Apache-2.0` `Java`
 - [Espial](https://github.com/jonschoning/espial) - An open-source, web-based bookmarking server. `AGPL-3.0` `Haskell`
 - [Firefox Account Server](https://mozilla-services.readthedocs.io/en/latest/howtos/run-fxa.html) - This allows you to host your own Firefox accounts server. ([Source Code](https://github.com/mozilla/fxa)) `MPL-2.0` `Nodejs, Java`
-- [Firefox Sync Server](https://github.com/mozilla-services/syncserver) - Sync Firefox bookmarks, passwords, history, tabs, preferences. `MPL-2.0` `Python`
+- [Firefox Sync Server](https://mozilla-services.readthedocs.io/en/latest/howtos/run-sync-1.5.html) - Sync Firefox bookmarks, passwords, history, tabs, preferences. `MPL-2.0` `Python`
```

Comment:
> Any software project you are adding to the list is actively maintained.

> Note that this repository is no longer being maintained. Use this at your own risk, and with the understanding that it is not being maintained, work is being done on its replacement, and that no support or assistance will be offered.

If it is no longer maintained, it should be removed completely from the list.

Reply:
I agree with this. It should be removed rather then updated, as Mozilla pointed out in that statement a replacement is coming but for now it's safe to remove.

Reply:
Got the line removed, but after a little bit more digging, I found this repo: https://github.com/mozilla-services/syncstorage-rs

I haven't tried to run it yet, but it appears to be the new replacement. Would it make sense to link to the new repo instead of removing the entry?

Reply:
It seems this is a successor to the firefox sync server, so i'd say just create the new line :)

Reply:
It would be nice to have some kind of confirmation that it
- is installable
- runs
- can effectively be used to sync firefox data

Reply:
Ah, valid arguments as I haven't thought of that. I might just test it as i''m currently running firefox, so I might spin up a instance and report back.

Reply:
Update: it takes a bit to compile along with a list of dependencies. and I'm now stuck on one issue which involves adding symbols and DSO to compile for rust. Docs are limiting but it is theoretically possible to compile this, but I haven't yet gotten into testing syncing firefox data due to compiling. There is a docker build but the dependency is a closed source library by Google cloud right now.  If anyone else can give this repo a shot, it would be most helpful. Thanks :)

Reply:
> theoretically possible to compile

:) Not really "production-ready" then?

> There is a docker build but the dependency is a closed source library by Google cloud right now

These ones are open-source if that's what you mean https://pypi.org/project/google-cloud-spanner/1.3.0/ https://github.com/mozilla-services/google-cloud-rust.

Anyway the priority is to remove the old, unsupported project. A new issue could be opened to add [syncstorage-rs](https://github.com/mozilla-services/syncstorage-rs), if/when it works. It would be good to have since there is currently no good self-hosted solution for Firefox Sync.

Thanks again